Demo/AuthenticatorDemo/ViewController.swift

```diff
@@ -7,24 +8,43 @@ class ViewController: UIViewController {
 
     /// Add `CellConfiguration` items to add new actionable rows to the table view in `ViewController`.
     lazy var configuration: [CellConfiguration] = [
-        CellConfiguration(text: "Show Login") { [weak self] in
+        CellConfiguration(text: "Show Login - With Google SDK") { [weak self] in
             guard let self else { fatalError() }
+
+            self.initializeWordPressAuthenticator(withoutGoogleSDK: false)
+            WordPressAuthenticator.showLoginFromPresenter(self, animated: true)
+        },
+        CellConfiguration(text: "Show Login - Without Google SDK") { [weak self] in
+            guard let self else { fatalError() }
+
+            self.initializeWordPressAuthenticator(withoutGoogleSDK: false)
             WordPressAuthenticator.showLoginFromPresenter(self, animated: true)
+        },
+        CellConfiguration(text: " Google SignIn - Standalone, Wihout SDK") { [weak self] in
+            guard let self else { fatalError() }
+
+            self.initializeWordPressAuthenticator(withoutGoogleSDK: false)
```

Comment:
The argument value here should also be true?

Reply:
🤦‍♂️

Reply:
Address in 80072fd8, while also using more straightforward "with Google SDK".